libs/application/core/src/types/Fields.ts

```diff
@@ -96,6 +96,7 @@ export interface SelectField extends BaseField {
   readonly type: FieldTypes.SELECT
   component: FieldComponents.SELECT
   options: MaybeWithApplication<Option[]>
+  defaultValue?: Option
```

Comment:
nice, I like this. We will definitely need to add `defaultValue` to many other fields later on, maybe even `BaseField`